indico/util/user.py

```diff
@@ -88,6 +97,17 @@ def principal_from_fossil(fossil, allow_pending=False, allow_groups=True, allow_
         if group.group is None and not allow_missing_groups:
             raise ValueError('Multipass group does not exist: {}:{}'.format(provider, id_))
         return group
+    elif category:
+        category_roles = list(
+            itertools.chain.from_iterable(
+                c.roles for c in category.chain_query.options(joinedload("roles"))
```

Comment:
we usually use single quotes except for english strings that contain `'` in the string itself (or any other cases where using single quotes would require more escaping)